Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.h

```diff
@@ -160,6 +163,8 @@ class RemoteLayerTreeEventDispatcher : public ThreadSafeRefCounted<RemoteLayerTr
 
     std::unique_ptr<RunLoop::Timer> m_delayedRenderingUpdateDetectionTimer;
 
+    HashSet<Ref<RemoteAcceleratedEffectStack>> m_effects;
```

Comment:
Also not sure if we should store these here. Can we just store a set of nodes that have effects that we need to pay attention to?

In general, try to avoid storing data in multiple places.

Reply:
Cc @mattwoodrow here to answer as well.

Reply:
In the latest patch this class now has `HashMap<WebCore::PlatformLayerIdentifier, Ref<RemoteAcceleratedEffectStack>> m_effectStacks;` as the unique place where the effects are stored for macOS.